source/source.go

```diff
@@ -0,0 +1,49 @@
+package source
+
+import (
+	"context"
+	"fmt"
+	"github.com/AdamHaffar/conduit-connector-airtable/config"
+
+	sdk "github.com/conduitio/conduit-connector-sdk"
+)
+
+type Source struct {
+	sdk.UnimplementedSource
+
+	config           config.Config
+	lastPositionRead sdk.Position
+}
+
+func NewSource() sdk.Source {
+	return &Source{}
+}
+
+func (s *Source) Configure(ctx context.Context, cfg map[string]string) error {
+	sdk.Logger(ctx).Info().Msg("Configuring a Source Connector...")
+	SourceConfig, err := config.ParseBaseConfig(cfg)
+	if err != nil {
+		return fmt.Errorf("couldn't parse the source config: %w", err)
+	}
+
+	s.config = SourceConfig
+	sdk.Logger(ctx).Info().Msg("Successfully configured the source connector")
+
+	return err
```

Comment:
IMHO, it would be better to simply have `return nil` here. Even though we're pretty much it's nil here, using `return nil` makes it really clear that no error happened.